integration_tests/class_01.py

```diff
@@ -0,0 +1,28 @@
+from lpython import i32,i64
+
+def fn_2():
+    print("Inside fn_2")
+    return
+ 
+class Test:
+    mem : i64 = i64(5)
+    s   : str = "abc"
+    def fn_1():
+        print("Inside fn_1")
+        print("fn_2 called")
+        fn_2()
+        print("fn_3 called")
+        Test.fn_3()
+        return 
```

Comment:
```suggestion
        return
```
No extra spaces as well